src/Symfony/Bridge/Doctrine/Form/ChoiceList/ORMQueryBuilderLoader.php

```diff
@@ -59,8 +59,8 @@ public function __construct($queryBuilder, $manager = null, $class = null)
         if ($queryBuilder instanceof \Closure) {
             @trigger_error('Passing a QueryBuilder closure to '.__CLASS__.'::__construct() is deprecated since version 2.7 and will be removed in 3.0.', E_USER_DEPRECATED);
 
-            if (!$manager instanceof EntityManager) {
-                throw new UnexpectedTypeException($manager, 'Doctrine\ORM\EntityManager');
+            if (!$manager instanceof EntityManagerInterface) {
```

Comment:
I guess so, however this is affecting me on 2.6/2.7 so even if not applied to master it should be backported to those versions - I'll look at testing against both interfaces to maintain BC.